roadmap/implementers-guide/src/types/runtime.md

```diff
@@ -35,7 +35,7 @@ struct HostConfiguration {
 	/// The amount of blocks ahead to schedule parathreads.
 	pub scheduling_lookahead: u32,
 	/// Total number of individual messages allowed in the parachain -> relay-chain message queue.
-	pub max_upward_queue_count: u32,
+	pub max_upward_queue_capacity: u32,
```

Comment:
Nit: `max_upward_queue_capacity` suggests that queue capacity is adjustable, and this is the max legal value. Either `max_upward_queue_count` or `upward_queue_capacity` would be appropriate IMO, but I think `max_upward_queue_capacity` changes the meaning a bit.